src/adzerk/response_models.rs

```diff
@@ -214,7 +214,12 @@ fn get_cdn_image(full_image_path: &str) -> Result<String, ProxyError> {
 }
 
 fn get_domain_affinities(name: Option<String>) -> &'static HashMap<String, u32> {
-    name.and_then(|name| defaults::DOMAIN_AFFINITIES.get(&name))
+    let affinities: &HashMap<_, _> = if cfg!(test) {
```

Comment:
Note that `cfg!()` evaluates to a Boolean value at compile time, so the optimizer will only keep one branch of this `if`, and the code won't branch at runtime.

Reply:
oh that's interesting